src/client/components/vision2/camera/line_projection.ts

```diff
@@ -0,0 +1,105 @@
+import * as THREE from 'three'
+
+import { Vector2 } from '../../../math/vector2'
+import { Vector3 } from '../../../math/vector3'
+import { Vector4 } from '../../../math/vector4'
+import { mesh } from '../../three/builders'
+import { planeGeometry } from '../../three/builders'
+import { rawShader } from '../../three/builders'
+import { shaderMaterial } from '../../three/builders'
+import { Canvas } from '../../three/three'
+
+import { Lens } from './model'
+import fragmentShader from './shaders/line_projection.frag'
+import vertexShader from './shaders/line_projection.vert'
+
+export interface ConeSegment {
+  /** The camera space central axis of the cone. Only needed for a plane if the start and end are (anti-)parallel. */
```

Comment:
This comment now makes no sense
```suggestion
  /** The camera space central axis of the cone. */
```